homeassistant/components/isy994/config_flow.py

```diff
@@ -26,16 +27,21 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+UDN_UUID_PREFIX = "uuid:"
```

Comment:
Should these be moved into `const.py` and/or used as magic strings in the tests?

Reply:
Moving to `const.py` is cleaner.  Done and adjusted the tests